modules/shared/src/main/scala/org/tessellation/currency/dataApplication/package.scala

```diff
@@ -134,6 +136,8 @@ trait DataApplicationL0Service[F[_], D <: DataUpdate, DON <: DataOnChainState, D
     extends DataApplicationService[F, D, DON, DOF]
     with DataApplicationContextualOps[F, D, DON, DOF, L0NodeContext[F]] {
   def genesis: DataState[DON, DOF]
+
+  def onSnapshotConsensusResult(snapshot: Signed[CurrencyIncrementalSnapshot])(implicit A: Applicative[F]): F[Unit] = A.unit
```

Comment:
@ryle-goehausen just to confirm, is this where you want me to eventually trigger the snapshot push to the external EVM?